Service.Twitch/TwitchChatClient.cs

```diff
@@ -10,9 +10,11 @@
 using System.Text.RegularExpressions;
 using System.Threading;
 
+using ConstantAlias = Service.Core.Constants;
+
 namespace Service.Twitch
 {
-
+    ConstantAlias.ChatService ChatName = new ConstantAlias.ChatService();
```

Comment:
We should let the dependency injection (DI) container handle all the class instance creation. You don't need to instantiate the class, but just reference the members through the static class. (ie. ConstantAlias.ChatService.ChannelName). Please make necessary updates.